src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int COLOR_COUNT = Color.values().length;
+    private Random random = new Random();
+
     public String getRandomColor() {
```

Comment:
According to Java best practices, it's recommended to use access modifiers explicitly. Consider defining the access level for the `getRandomColor` method explicitly if you have a specific intention for package-private access level.